Makefile

```diff
@@ -74,3 +81,8 @@ register: _requires-sandbox-up  ## Register Flyte cookbook workflows
 fast_register: _requires-sandbox-up  ## Fast register Flyte cookbook workflows
 	$(call LOG,Fast registering example workflows in cookbook/$(EXAMPLES_MODULE))
 	$(call RUN_IN_SANDBOX,make -C cookbook/$(EXAMPLES_MODULE) fast_register)
+
+.PHONY: setup-kubectl
+kubectl-config: 
+	# In shell/bash, run: `eval $(make kubectl-config)`
+	echo "export KUBECONFIG=$(KUBECONFIG):~/.kube/config:$(KUBE_CONFIG)/k3s/k3s.yaml"
```

Comment:
should we nix the first $(KUBECONFIG)?  otherwise it'll just get longer and longer.  also why the circuitous route with the eval?  also, how does the `:` work?  if you `use-context xyz` it'll just go through in order until it finds an `xyz`?

Reply:
synced offline